keyboards/converter/ibm_terminal/keymaps/dsanchezseco/rules.mk

```diff
@@ -0,0 +1,22 @@
+# Build Options
```

Comment:
You only need the options that are being changed, here, really.

Reply:
fixed